package.json

```diff
@@ -1,18 +1,22 @@
 {
   "private": true,
   "scripts": {
+    "bad-linebreaks": "node ./scripts/bad-linebreaks.mjs",
+    "bad-linebreaks:fix": "npm run bad-linebreaks fix",
     "check-delegates": "node -e 'import(\"./scripts/check-delegates.mjs\").then(cd => cd.checkDelegates())'",
     "check-delegates-test": "node ./scripts/check-delegates-test.mjs",
+    "fix:all": "npm run lint:fix && npm run mdlint:fix && npm run bad-linebreaks:fix",
     "lint": "eslint . --ext .js,.mjs,.cjs",
     "lint:fix": "npm run lint -- --fix",
     "//": "markdownlint commands ignore old files",
     "mdlint": "markdownlint-cli2 '**/*.md' '!node_modules' '!meetings/201*/*.md' '!meetings/202[0-2]*/*.md' '!meetings/2023-0[1-3]/*.md'",
     "mdlint:fix": "markdownlint-cli2-fix '**/*.md' '!node_modules' '!meetings/201*/*.md' '!meetings/202[0-2]*/*.md' '!meetings/2023-0[1-3]/*.md'",
     "//": "markdownlint most likely to fail, so run that first",
-    "test": "npm run mdlint && npm run lint && npm run check-delegates-test && npm run check-delegates"
+    "test": "npm run mdlint && npm run bad-linebreaks && npm run lint && npm run check-delegates-test && npm run check-delegates"
```

Comment:
```suggestion
    "pretest": "npm run mdlint && npm run bad-linebreaks && npm run lint",
    "test": "npm run check-delegates-test && npm run check-delegates"
```

Reply:
sir...

"my ideal is that eventually npm test is sufficient to run every CI check" -Jordan Harband

Reply:
yes, and `npm test` automatically runs `npm run pretest` before it and `npm run posttest` after it (that's how all npm scripts work)

Reply:
ah that's right.. I forget this regularly...  but I'm also a fan of being explicit about what's happening and not relying on hidden magic (even if common knowledge)